### DISCUSSION TITLE
How to suppress prints?

### DISCUSSION THREAD

**Discussion by openSourcerer9000** [ultralytics/ultralytics#8867] [Q&A] [answered]
How to suppress it from printing during inference? I'm running millions of times on tiny patches, so my console just fills with this:

0: 256x256 (no detections), 6.5ms
Speed: 0.0ms preprocess, 6.5ms inference, 1.0ms postprocess per image at shape (1, 3, 256, 256)

**Comment by pderrenger** [comment, marked as answer]
To suppress prints during inference, you can set `verbose=False` in `model.predict()`.
